Providers/Modules/Plugins/SecurityBaseline/conf/security_baseline.conf

```diff
@@ -3,14 +3,14 @@
 <source> 
     type exec
     tag oms.security_baseline
-    command sleep 60 && /opt/microsoft/omsagent/plugin/omsbaseline -d  /etc/opt/microsoft/omsagent/conf/omsagent.d/ 
+    command sleep 60 && /opt/microsoft/omsagent/plugin/omsbaseline -d  /opt/microsoft/omsagent/plugin
```

Comment:
Can you explain why an accompanying PR is not being made in the OMSAgent repo to [this file](https://github.com/Microsoft/OMS-Agent-for-Linux/blob/master/installer/conf/omsagent.d/security_baseline.conf)?